ipatool

```diff
@@ -359,6 +363,14 @@ class Context(object):
                     repo_to=self.config['pagure-repository']
                 )
 
+        try:
+            self.bugzilla = bugzilla.Bugzilla(
+                url='https://bugzilla.redhat.com',
+                api_key=self.config['bz-token'],
+            )
+        except KeyError:
+            self.die(
+                'Attribute "bz-token" is missing, add it your config file.')
```

Comment:
Is it worth adding a BZ fetch here to validate that the API key is correct? A hardcoded BZ would do, it would just require a round-trip on every push that may or may not be necessary.

Reply:
Calling `bugzilla.logged_in` to verify that. I think it's worth it, but it adds a network call that might not be always required. 

`BugzillaError` was expected but `xmlrpclib.Fault` was raised, that's why I'm handling `Exception`.